uaclient/security.py

```diff
@@ -532,29 +679,11 @@ def prompt_for_affected_packages(
                 else:
                     print(status.MESSAGE_SECURITY_UPDATE_INSTALLED)
     if not any([upgrade_packages, upgrade_packages_ua]):
-        if is_fixable:
-            print(
-                status.MESSAGE_SECURITY_ISSUE_RESOLVED.format(issue=issue_id)
-            )
-        else:
-            print(
-                status.MESSAGE_SECURITY_ISSUE_NOT_RESOLVED.format(
-                    issue=issue_id
-                )
-            )
+        print(fix_message)
```

Comment:
Why are we dropping the `is_fixable` logic here ?  If we remove it, we will never shown that a USN/CVE is not fixed.

Also, this will impact our integration tests that do expect that USN/CVE to be marked as failed

Reply:
@lucasmoura not dropping it. It looked like it was just a bool to determine what message to print(resolved vs unresolved). So I just changed it to a str type so that it just contained the intended resolved/unresolved message. That allows us to avoid the duplicated if is_fixable: print(resolved) else: print(unresolved) in two places.